### ISSUE TITLE
Separate Keys for Line of Sight and Continue Movement Path

### ISSUE THREAD

**Issue by nemuc** [ja2-stracciatella/ja2-stracciatella#797]
Currently, the END key has two conflicting uses

1. To make the merc **continue** moving on his way
2. To show **LINE of SIGHT**

The main conflictshappens in this use case:

1. - Merc moves in turn-based mode
2. - Merc stops due to the sighting of a new enemy

If the player now presses END to analyze the LINE of SIGHT, the merc instead continues his movement.

I thus propose to have two different keys for these two different actions.

**Comment by lynxlynxlynx** [comment]
Were the same ones used in vanilla? If so, we'll need to config this option, whatever it may be. I suggest we leave both as is, except require two presses of END (emulating a double click) in this scenario for continuation to happen.

**Comment by nemuc** [comment]
Yes this is from vanilla. I second the suggestion for the two presses of END.
